src/normalizations/cms/useNormalizedCMSResults.js

```diff
@@ -20,11 +20,73 @@ export const EDITORIAL_FIELD_TYPE_VALUES = {
 }
 
 // Drupal JSONapi encodes `&` in URLs, which React can't handle https://github.com/facebook/react/issues/6873#issuecomment-227906893
-function cleanupDrupalUri(uri) {
-  return uri.replace(/&amp;/g, '&')
+const cleanupDrupalUri = (uri) => uri.replace(/&amp;/g, '&')
+
+export const getLocaleFormattedDate = ({
+  field_publication_date,
+  field_publication_day,
+  field_publication_year,
+  field_publication_month,
+} = {}) => {
+  const year = parseInt(field_publication_year, 10)
+  const yearIsValidNumber = Number.isNaN(year) === false
```

Comment:
Would negating this boolean not be shorter?

```suggestion
  const yearIsValidNumber = !Number.isNaN(year)
```

Reply:
Definitely shorter, but not necessarily more readable.

Reply:
It's being used in other parts of the code like this already though (see the `dateParts` variable).